tests/feature/test_feature_base_dir.py

```diff
@@ -5,44 +5,44 @@
 
 
 @pytest.mark.parametrize("base_dir", NOT_EXISTING_FEATURE_PATHS)
-def test_feature_path_not_found(testdir, base_dir):
+def test_feature_path_not_found(pytester, base_dir):
     """Test feature base dir."""
-    prepare_testdir(testdir, base_dir)
+    prepare_pytester(pytester, base_dir)
 
-    result = testdir.runpytest("-k", "test_not_found_by_ini")
+    result = pytester.runpytest("-k", "test_not_found_by_ini")
     result.assert_outcomes(passed=2)
 
 
-def test_feature_path_ok(testdir):
+def test_feature_path_ok(pytester):
     base_dir = "features"
-    prepare_testdir(testdir, base_dir)
+    prepare_pytester(pytester, base_dir)
 
-    result = testdir.runpytest("-k", "test_ok_by_ini")
+    result = pytester.runpytest("-k", "test_ok_by_ini")
     result.assert_outcomes(passed=2)
 
 
-def test_feature_path_by_param_not_found(testdir):
+def test_feature_path_by_param_not_found(pytester):
     """As param takes precedence even if ini config is correct it should fail
     if passed param is incorrect"""
     base_dir = "features"
-    prepare_testdir(testdir, base_dir)
+    prepare_pytester(pytester, base_dir)
 
-    result = testdir.runpytest("-k", "test_not_found_by_param")
+    result = pytester.runpytest("-k", "test_not_found_by_param")
     result.assert_outcomes(passed=4)
 
 
 @pytest.mark.parametrize("base_dir", NOT_EXISTING_FEATURE_PATHS)
-def test_feature_path_by_param_ok(testdir, base_dir):
+def test_feature_path_by_param_ok(pytester, base_dir):
     """If ini config is incorrect but param path is fine it should be able
     to find features"""
-    prepare_testdir(testdir, base_dir)
+    prepare_pytester(pytester, base_dir)
 
-    result = testdir.runpytest("-k", "test_ok_by_param")
+    result = pytester.runpytest("-k", "test_ok_by_param")
     result.assert_outcomes(passed=2)
 
 
-def prepare_testdir(testdir, ini_base_dir):
-    testdir.makeini(
+def prepare_pytester(pytester, ini_base_dir):
```

Comment:
This should still be called prepare_testdir

Reply:
Done.